python/grass/projpicker/Makefile

```diff
@@ -0,0 +1,20 @@
+MODULE_TOPDIR = ../../..
+
+include $(MODULE_TOPDIR)/include/Make/Other.make
+include $(MODULE_TOPDIR)/include/Make/Python.make
+
+DSTDIR = $(ETC)/python/grass/projpicker
+
+MODULES = projpicker common coor_latlon coor_xy
+
+PYFILES := $(patsubst %,$(DSTDIR)/%.py,$(MODULES) __init__)
+PYCFILES := $(patsubst %,$(DSTDIR)/%.pyc,$(MODULES) __init__)
+DSTFILES := $(DSTDIR)/VERSION $(DSTDIR)/projpicker.db
```

Comment:
Perhaps include a readme with documentation on what this file is and how to update it.

Reply:
README added in 0208123